src/plugins/irc-events/invite.js

```diff
@@ -0,0 +1,29 @@
+var _ = require("lodash");
+var Msg = require("../../models/msg");
+
+module.exports = function(irc, network) {
+	var client = this;
+	irc.on("invite", function(data) {
+		var target = data.to;
+		if (target.toLowerCase() === irc.me.toLowerCase()) {
```

Comment:
Fair point @xPaw but as an aside, do you think this should be changed on all events (in a different PR of course)?